common/unsafe/src/test/java/org/apache/spark/unsafe/hash/Murmur3_x86_32Suite.java

```diff
@@ -51,6 +53,23 @@ public void testKnownLongInputs() {
     Assert.assertEquals(-2106506049, hasher.hashLong(Long.MAX_VALUE));
   }
 
+  // SPARK-23381 Check whether the hash of the byte array is the same as another implementations
```

Comment:
We could add a randomized (disabled) test here. That might increase the confidence we have in the new hash.

Reply:
Yeah, we can do it in the follow-up PR.